### PR TITLE
Allow direct media to be stickerized without caption

### PR DESCRIPTION
**Pull request by derhnyel**

### CODE REVIEW

**Review by derhnyel — COMMENTED** [review]
Checks if message is from a group before stickerizing

**Comment by deven96** [comment]
More like allows direct media to be stickerized without caption

File: main.go [hunk 1]
```diff
@@ -73,15 +73,16 @@ func eventHandler(evt interface{}) {
...
 		videoMatch := captionIsCommand(eventInfo.Message.GetVideoMessage().GetCaption())
 		// check if quoted message with correct caption references media
 		quotedMatch := captionIsCommand(quotedText) &&
 			(quotedImage != nil || quotedVideo != nil)
-
-		if imageMatch || videoMatch || quotedMatch {
-			if quotedMatch {
+		isPrivateMedia := (quotedImage != nil || quotedVideo != nil) && !groupMessage
+		if imageMatch || videoMatch || quotedMatch || isPrivateMedia {
+			if quotedMatch || isPrivateMedia {
```
Comment (deven96) [inline main.go:85]: you don't need to add isPrivateMedia on this line... this line is for when someone quotes a previously sent media

**Review by deven96 — CHANGES_REQUESTED** [review]
Final change

**Pull request merged**